doc/releases/nautilus.rst

```diff
@@ -216,13 +216,24 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
-#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
-   that ceph-volume can activate them after a host reboot.  On each
-   host containing OSDs,::
+#. If there are any OSDs in the cluster deployed with ceph-disk (e.g.,
```

Comment:
Can we print that in the package install logs?

Reply:
What do you mean by install logs? If you mean an rpm post script then the packages can't (well, technically they can but it is against the packaging policies) print anything when running the post scripts.